nncf/openvino/graph/metatypes/openvino_metatypes.py

```diff
@@ -191,10 +191,21 @@ class OVSubtractMetatype(OVOpMetatype):
     hw_config_names = [HWConfigOpName.SUBTRACT]
 
 
+@OV_OPERATOR_METATYPES.register()
+class OVSQMultiplyMetatype(OVOpMetatype):
```

Comment:
This is not good idea to introduce new metatype for a specific algorithm. I hope you can implement this check "if "nncf_smooth_quant" in node.get_friendly_name():" into algorithm.

Reply:
The original problem looks like this: after the SmoothQuant inserts scales, we need to avoid these scale quantizations.
I've solved this problem by introducing a new type that handles as agnostic for insertion but blocks FakeQuantize propagation.
I wouldn't implement checking `nncf_smooth_quant` in the MinMax algorithm since algorithms are independent.
Do you have any other ideas to solve the problem?

Reply:
You already introduced implicit dependent via nncf graph. You check `nncf_smooth_quant` name during building nncf graph. I would recommend explicit check this condition into the algorithm and add all such node into ignored scope for the propagation solver algorithm.

Reply:
It's not implicitly dependent, since NNCFGraph is another entity.
Checking `nncf_smooth_quant` will break the statement that algorithms don't know about each other and can exist independently, this is not a solution.

Reply:
Done as discussed.